app/controllers/admin/assortments_controller.rb

```diff
@@ -48,4 +48,8 @@ def assortments
     @assotrments = Assortment.all
   end
 
+  def assortment_params
+    params.require(:assortment).permit(:name, :category_id, category_attributes: [:name])
```

Comment:
why do you need category_attributes: [:name] ?
if you want to create category through assortment, you have to add accepts_nested_attributes_for to assortment, but i don't think that we need to do that